experimental/packages/opentelemetry-exporter-otlp-http/src/transformMetrics.ts

```diff
@@ -29,17 +29,69 @@ import { toCollectorResource } from './transform';
 import { OTLPExporterConfigBase, opentelemetryProto } from './types';
 
 /**
- * Converts labels
- * @param labels
+ * Converts attributes to string-string key value pairs.
+ * Note that these Labels are deprecated in OpenTelemetry.
+ * @param attributes
  */
 export function toCollectorLabels(
-  labels: Labels
+  attributes: Attributes
 ): opentelemetryProto.common.v1.StringKeyValue[] {
-  return Object.entries(labels).map(([key, value]) => {
+  return Object.entries(attributes).map(([key, value]) => {
     return { key, value: String(value) };
   });
 }
 
+export function toCollectorAttributes(
```

Comment:
Do we have a unit test for this new method?

Reply:
This is a fairly large method. Maybe it should be broken into smaller parts?